### ISSUE TITLE
Why does _union() start with an underscore in stdlib.h?

### ISSUE THREAD

**Issue by virtualritz** [libfive/libfive#433]
See subject.

**Comment by doug-moen** [comment]
`union` is a keyword in C/C++, so the name `union()` would not work.

**Comment by mkeeter** [comment]
Not quite – this is because the C and C++ `union` need to be distinguished by the bindings generator, and functions which take zero arguments but return different types aren't allowed.

**Comment by virtualritz** [comment]
Yeah, I get it. I wrote a `gen_rust.py` and there I have the issue with `move` (Rust keyword). I swap that out for `moveit`.
